tools/packaging/rpm/istio/istio.spec

```diff
@@ -37,9 +37,7 @@ Source0:        istio.tar.gz
 #Source1:        istiorc
 #Source2:        buildinfo
 Source3:        istio-start.sh
-Source4:        istio-node-agent-start.sh
 Source5:        istio.service
-Source6:        istio-auth-node-agent.service
 
 # e.g. el6 has ppc64 arch without gcc-go, so EA tag is required
 ExclusiveArch:  %{?go_arches:%{go_arches}}%{!?go_arches:%{ix86} x86_64 aarch64 %{arm}}
```

Comment:
ExclusiveArch guidelines are here:
https://docs.fedoraproject.org/en-US/packaging-guidelines/#_arch_specific_runtime_and_build_time_dependencies

I think all this go_arches stuff seems nonsensical from a packaging point of view. Can you please sort out a proper ExclusiveArch line (or remove it entirely as per guidelines).

Reply:
Please follow the fedora packaging guidelines for this spec file: https://docs.fedoraproject.org/en-US/packaging-guidelines/